src/Illuminate/Queue/BeanstalkdQueue.php

```diff
@@ -148,7 +148,7 @@ public function deleteMessage($queue, $id)
      */
     public function getQueue($queue)
     {
-        return $queue ?: $this->default;
+        return $this->getQueuePrefix().($queue ?: $this->default);
```

Comment:
I think there would be better to create method in parent class, like `getQueueName($queue)`, and call it in all driver classes.